ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml

```diff
@@ -33,165 +35,139 @@ StatusSectionLayout {
     QtObject {
         id: d
 
-        property string searchText: ""
-        property int layoutVMargin: 70
-        property int layoutHMargin: 64
-        property int titlePixelSize: 28
-        property int subtitlePixelSize: 17
+        // values from the design
+        readonly property int layoutTopMargin: 10
+        readonly property int layoutBottomMargin: 249
+        readonly property int layoutHMargin: 64
+        readonly property int layoutWidth: 1037
+        readonly property int titlePixelSize: 28
+        readonly property int preventShadowClipMargin: 40
+
+        readonly property bool searchMode: searcher.text.length > 0
 
         function navigateToCommunity(communityId) {
             root.communitiesStore.setActiveCommunity(communityId)
         }
     }
 
+    SortFilterProxyModel {
+        id: searchModel
+
+        sourceModel: root.communitiesStore.curatedCommunitiesModel
+
+        filters: ExpressionFilter {
+            enabled: d.searchMode
+            expression: {
+                searcher.text
```

Comment:
Reevaluation?

Reply:
Yes, `ExpressionFilter` detects dependencies in a specific way and it causes that tricks like that are needed.